### ISSUE TITLE
pgeo.Path does not allow paths with less than 2 points.

### ISSUE THREAD

**Issue by despondency** [volatiletech/sqlboiler#994]
The problem is the following:

We are using the geometric type Path from PostgreSQL, to persist a bunch of points on a path. However if we have Point(A) == Point(B) the path between them would be a single Point(A==B).

We insert the data in the DB where we have a path with one point, however when we try to get it, we get the wrong path exception. Why is this assumption made on ORM level? Why does sqlboiler allows us to persist one point on a path, but never to get it? 

Much thanks for the assistance.

The code path that contains the problem is this:

pgeo.Path 

```
(*p).Points, err = parsePoints(val)
	if err != nil {
	return err
}

if len((*p).Points) < 2 {
     return errors.New("wrong path")
}
```

**Comment by aarondl** [comment]
I actually have no idea why this is like this as I had nothing to do with the implementation of it and would rely on an expert in pgeo to tell us why that might be the case.

**Comment by saulortega** [comment]
I wrote that code, and I assumed that a path should have at least two points, but you are right, it can have one single point.

Fixed in https://github.com/volatiletech/sqlboiler/pull/1123

**Comment by stephenafamo** [comment]
Released in `v4.11.0`

**Comment by stephenafamo** [comment]
Thanks for the contribution @saulortega
